SKTTLVParser.m

```diff
@@ -0,0 +1,52 @@
+//
+//  SKTTLVParser.m
+//  Capture
+//
+//  Created by Chrishon Wyllie on 4/3/20.
+//  Copyright © 2020 SocketMobile. All rights reserved.
+//
+
+#import "SKTTLVParser.h"
+
+@implementation SKTTLVParser
+-(id) init {
+    self = [super init];
+    if (self) {
+    }
+    return self;
+}
+
++(NSDictionary*)parseDataForTLVFormat:(NSData *)decodedDataWithoutPCBHeader {
```

Comment:
I would have preferred something even more generic, maybe something like `tlvData` instead of `decodedDataWithoutPCBHeader`?